incubator/logstash/Chart.yaml

```diff
@@ -15,3 +15,5 @@ maintainers:
   email: jrodgers@powerhrg.com
 - name: christian-roggia
   email: christian.roggia@gmail.com
+- name: josdotso
```

Comment:
Please clarify why you added yourself as maintainer. You will be expected to continue contributing and review PRs.

@christian-roggia Are you fine with this?

Reply:
Thanks @unguiculus!  This was my rookie mistake. I hope to continue contributions to this and other charts in the repo, but I'd should become more comfortable with the patterns and processes before committing to being a binding voice in reviews.  Once I have some more PRs behind me, perhaps we can revisit?

cc: @christian-roggia